### PR TITLE
Convert migration PDF to HTML to aid contribution

### PR DESCRIPTION
**Pull request by SueChaplain**
Create an HTML page for the migration content so that
individual contributors can update it with progress for
the different components that are being worked on.
E.g. IcedTea-web, fonts, OpenJFX

Minor changes to other pages that mention migration.

Signed-off-by: Sue Chaplain <sue_chaplain@.uk.ibm.com>

##### Checklist

- [ ] `npm test` passes
- [x] documentation is changed or added
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by SueChaplain** [comment]
Hi @karianna  - Quite a few changes here, which I've detailed. We briefly discussed this being HTML in a slack message, so I went ahead and did this work. Happy to make any changes needed
if I've overstepped the remit :-)

Home page
- Added a new link to Migration at the bottom of the page

Menu
- Renamed Migration Guide to Migration to create 3 clear areas under the menu build pages: Installation, Migration, Support.

Migration Guide
- Created an HTML Migration page to replace the PDF so that everyone can contribute to keep it up to date as work progresses on various things like IcedTea-Web and Fonts.
- Did not re-create everything that was in the Migration guide as some areas looked to be duplicated elsewhere on the site:
	- What is AdoptOpenJDK?, Oracle's Java Support Has Changed: mostly duplicated on the About page and Support page
	- Which platforms are supported?: replaced with a single sentence in the overview to direct users to the Platform Support Matrix
	- How Can I Install AdoptOpenJDK?: Not needed as Installation is on the main menu
	- Who Can I Contact For Commercial Support?, duplicated on the Support page
	
Installation
- Removed the section about Enabling JavaFX / OpenJFX via IcedTea-Web , putting the link to the video into the Migration page under IcedTea-Web.
- Added a new section for Migrating

Support
- Removed the link to the migration guide because the Migration section is clearly listed in the menu.

File: src/handlebars/installation.handlebars [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <h1 class="large-title">Installation</h1>
 
-  <h2><a href="./MigratingtoAdoptOpenJDKfromOracleJava.pdf" target="_blank">Migration Guide</a></h2>
+  <h2><a href="./migration.html" target="_blank">Migration</a></h2>
```
Comment (karianna) [inline src/handlebars/installation.handlebars]: I think I prefer `Migration Guide` but it's not a hill I'll die on :-)

File: src/handlebars/installation.handlebars [hunk 2]
```diff
@@ -62,9 +62,10 @@
             </div>
 
             <div style="margin-top: 3.0em;">
-              <h2 class="bold">Enabling JavaFX / OpenJFX via IcedTea-Web</h2>
-              <p>If you're looking to add Java FX / OpenJFX Support you can currently do so via adding <a href="https://icedtea.classpath.org/wiki/IcedTea-Web">IcedTea-Web</a>.</p>
-              <p>Follow the instructions in this <a href="https://youtu.be/wRRS0LejZuM">IceaTea-Web Installation</a> video.</p>
+              <h2 class="bold">Migration</h2>
+              <p>If you're migrating to an (Adopt)OpenJDK, you can learn about the differences between an Oracle JDK and an OpenJDK in
```
Comment (karianna) [inline src/handlebars/installation.handlebars]: I think grammatically (and trademark wise) that `(Adopt)OpenJDK` needs to be `AdoptOpenJDK` and we should drop the `an` in front of AdoptOpenJDK and OpenJDK and Oracle JDK _or_ it needs to read `an XXX JDK binary`.  Hope that all makes sense?

File: src/handlebars/migration.handlebars [hunk 3]
```diff
@@ -0,0 +1,206 @@
+{{> header title='migration | ' }}
+
+<!-- TODO Style should match installation page, it doesn't quite... -->
+<main class="grey-bg">
+
+  <h1 class="margin-bottom large-title">Migration</h1>
```
Comment (karianna) [inline src/handlebars/migration.handlebars]: perhaps `Migration Guide`

File: src/handlebars/migration.handlebars [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+  <div class="align-left support">
+
+    <div class="anchor" onmouseover="document.getElementById('anchor1').style.visibility='visible'" onmouseout="document.getElementById('anchor1').style.visibility='hidden'"><a href="#community-support" class="anchor">
+      <img src="dist/assets/anchor.png" id="anchor1" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="migration-overview">Migrating to AdoptOpenJDK</a></h2>
+    </div>
+    <div class="margin-bottom">
+      <p>If you are moving from an Oracle JDK to an OpenJDK from the AdoptOpenJDK project, you'll probably have a few key questions on your mind. Firstly,
```
Comment (karianna) [inline src/handlebars/migration.handlebars]: same comments about the `an` usage

Comment (karianna) [reply src/handlebars/migration.handlebars]: And (Adopt)OpenJDK

File: src/handlebars/migration.handlebars [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+        you'll want to know what the differences are between an Oracle JDK and an (Adopt)OpenJDK and whether there are any steps you need to take to migrate.
+        If you've looked at the home page, you'll also know that we provide OpenJDKs with a choice of Java virtual machine. So, you might also be wondering
```
Comment (karianna) [inline src/handlebars/migration.handlebars]: Java Virtual Machine (JVM)

File: src/handlebars/migration.handlebars [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+        </tbody>
+      </table>
+      <p><i class="fa fa-pencil" aria-hidden="true"></i><span class="sr-only">Note:</span> JavaFX, JFR, and JMC were all contributed
+      by Oracle to the OpenJDK project for releases after Java 8.</p>
+
+      <h3 id="icedtea-web">IcedTea-web</h3>
+
+      <p>AdoptOpenJDK are working on a project to provide IcedTea-web as an open source replacement for Webstart and Java plugin technology. This feature
+      will be available via an installer option in AdoptOpenJDK binaries. In the meantime, this <a href="https://youtu.be/wRRS0LejZuM">IcedTea-Web installation video</a>
+    provides information on how to install it independently.</p>
```
Comment (karianna) [inline src/handlebars/migration.handlebars]: Whitespace

File: src/handlebars/migration.handlebars [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+    <div class="margin-bottom">
+      <p><a href="https://www.eclipse.org/openj9">Eclipse OpenJ9</a> is an alternative JVM that implements the <a href="https://docs.oracle.com/javase/specs/index.html">Java Virtual Machine Specification</a>.
+      You can think of it as a drop in replacement for the HotSpot VM, which is part of the reference implementation. Most Java applications should run
+      on an OpenJDK that contains the OpenJ9 VM without changing anything. However, because it is an independent implementation there are some slight
+      differences. You can learn about these differences by reading the <a href="https://www.eclipse.org/openj9/docs/openj9_newuser/">New to OpenJ9?</a> topic
+      in the OpenJ9 user documentation.</p>
+
+    </div>
+
+    <div class="anchor" onmouseover="document.getElementById('anchor3').style.visibility='visible'" onmouseout="document.getElementById('anchor3').style.visibility='hidden'"><a href="#commercial-support" class="anchor">
```
Comment (karianna) [inline src/handlebars/migration.handlebars]: anchor3 is a repeat here

File: src/handlebars/migration.handlebars [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+      <img src="dist/assets/anchor.png" id="anchor3" name="anchor" alt="anchor icon"></a>
+      <h2 class="bold"><a id="migration-resources">More resources</a></h2>
+    </div>
+
+    <div class="margin-bottom">
+      <p>Here are a few of the resources available to help you with migrating to an OpenJDK from the AdoptOpenJDK project:
```
Comment (karianna) [inline src/handlebars/migration.handlebars]: probably shorter to say `Migrating to AdoptOpenJDK`

**Comment by SueChaplain** [comment]
@karianna - thanks for the review. I think I have addressed all your points. Hope this is good now.

**Pull request merged**